controllers/result.go

```diff
@@ -0,0 +1,44 @@
+package controllers
+
+import (
+	"net/http"
+
+	"go-google-scraper-challenge/services/scraper"
+
+	"github.com/beego/beego/v2/server/web"
+)
+
+// ResultController operations for User
+type ResultController struct {
+	BaseController
+}
+
+// URLMapping map user controller actions to functions
+func (c *ResultController) URLMapping() {
+	c.Mapping("List", c.List)
+	c.Mapping("Create", c.Create)
+}
+
+func (c *ResultController) List() {
+	c.EnsureAuthenticatedUser(true)
+
+	c.Layout = "layouts/default.html"
+	c.TplName = "search_results/list.html"
+
+	web.ReadFromRequest(&c.Controller)
+}
+
+func (c *ResultController) Create() {
+	c.EnsureAuthenticatedUser(false)
+
+	keywords := []string{
```

Comment:
These are temporary keywords to test the scraper, and they will be removed and replace with keywords from the CSV file on #6